DevelopmentTools/HCALLMapDumper.cc

```diff
@@ -1197,20 +1201,21 @@ void HCALLMapDumper::makedbHOCalibLMapObject(std::string HCALLMapDbStr, std::str
 
   //Create Table in SQL
   //i(Unique key)
-  //Side Eta Phi dPhi Depth Det
+  //Side Eta Phi dPhi ch_type Name
   //RBX 
   //RM RM_FI FI_CH
-  //ppCol ppRow ppLC
-  //Crate HTR FPGA HTR_F
+  //ppCol ppRow ppCpl ppLC
+  //Crate HTR FPGA HTR_FI Spigot DCC
+  //QIE8id
 
   std::string CreateTable = "CREATE TABLE IF NOT EXISTS " + HOCalibTableStr + "(" \
                             "ID INT PRIMARY KEY NOT NULL, " \
-                            "Side INT NOT NULL, Eta INT NOT NULL, Phi INT NOT NULL, dPhi INT NOT NULL, Depth INT NOT NULL, Det TEXT NOT NULL, " \
```

Comment:
Do not change the schema name from Depth to ch_type. This will break the QIE table generation.